kornia/losses/psnr.py

```diff
@@ -0,0 +1,44 @@
+import torch
+import torch.nn as nn
+from torch.nn.functional import mse_loss
+
+
+class PSNR(nn.Module):
+    r"""Creates a criterion that calculates the PSNR between 2 images.
+
+    Arguments:
+        max_val (float): Maximum value of signal
+
+    Shape:
+        - signal: :math:`(*)`
+        - approximation: :math:`(*)` same shape as signal
+        - output: :math:`()` a scalar
+
+    Examples:
+        >>> kornia.losses.psnr(torch.ones(1), 1.2*torch.ones(1), 2)
+        tensor(20.0000) # 10 * log(4/((1.2-1)**2)) / log(10)
+
+    reference:
+        https://en.wikipedia.org/wiki/Peak_signal-to-noise_ratio#Definition
+    """
+
+    def __init__(self, max_val) -> None:
+        super(PSNR, self).__init__()
+        self.max_val = max_val
+
+    def forward(  # type: ignore
+            self, signal: torch.Tensor, approximation: torch.Tensor) -> torch.Tensor:
+        return psnr(signal, approximation, self.max_val)
+
+
+def psnr(signal: torch.Tensor, approximation: torch.Tensor, max_val) -> torch.Tensor:
```

Comment:
In order to be consistent with PyTorch I would rename the function / module name as `psnr_loss` / `PSNRLoss` . In addition, for same reason as before rename `signal -> input` , `approximation -> target`, we can describe in the documentation what's the meaning of each variable.

Reply:
Done